OracleWebCenterContent/dockerfiles/12.2.1.4/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+# 
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle WCCONTENT 12.2.1.4.0
+#
+# Base image of this dockerfile is Oracle FMW INFRASTRUCTURE 12.2.1.4.0 docker image.
+# Pull base image from the Oracle Container Registry or Docker Store and tag the name as 'oracle/fmw-infrastructure:12.2.1.4.0'
+# 
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) fmw_12.2.1.4.0_wccontent_generic.jar
+#
+# Download all dependent binaries from : http://changeiq.oraclecorp.com/fmw-stage/?release=FMW12.2.1.4.0
+# 
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put downloaded binary in the same directory as this Dockerfile
+# Run:
+#      $ docker build -t oracle/wccontent:12.2.1.4.0 .
+# 
+#
+
+# Pull base image from the Oracle Container Registry or Docker Store and tag the name as 'oracle/fmw-infrastructure:12.2.1.4.0'
+# ---------------
+FROM oracle/fmw-infrastructure:12.2.1.4.0
+
+# Maintainer
+# ----------
+MAINTAINER WBECENTER CONTENT DEV
+
+# -------------------------------------------------------------
+# Environment variables required for this build (do NOT change)
+# -------------------------------------------------------------
+USER root
+
+ENV CONTENT_JAR=fmw_12.2.1.4.0_wccontent.jar \
+    ORACLE_HOME=/u01/oracle \
+    VOLUME_DIR=/u01/oracle/user_projects \
+    SCRIPT_FILE=/u01/oracle/container-scripts/* \
+    USER_MEM_ARGS="-Djava.security.egd=file:/dev/./urandom" \
+    PATH=$PATH:$JAVA_HOME/bin:$ORACLE_HOME/oracle_common/common/bin:/u01/oracle/wlserver/common/bin:/u01/oracle/container-scripts
+    
+# -------------------------------------------------------------
+# Copy Java JDK files
+# -------------------------------------------------------------
+# ADD $JAVA_PKG /usr/java/
+
+# -------------------------------------------------------------
+# Install packages and adjust file permissions
+# -------------------------------------------------------------
+RUN env && \
+    mkdir -p /u01 && \
+    yum install -y libaio && \
+    yum install -y tar && \
+    yum install -y procps && \
+    yum install -y sudo hostname wget curl zip unzip && \
```

Comment:
Why are you installing `sudo` into a container? Also, why do you need both `wget` and `curl` considering they do the same thing. Note that `curl` is installed by default in all images anyway.